src/test/java/com/roisoftstudio/godutch/integration/login/paths/SignPathTest.java

```diff
@@ -89,7 +86,7 @@ public void signOut_shouldReturnOk_ifSignsOutBeingSignedIn() throws Exception {
         HttpRequest signInResponse = signInRequest(emailValue, "pass");
         assertThat(signInResponse.code()).isEqualTo(OK.getStatusCode());
 
-        String token = signInResponse.body();
+        String token = responseParamsToJSON(signInResponse).getString("token");
```

Comment:
here would be better to just pass the request.body() instead of the httprequest.  Or just 
  String token = new JSONObject(signInResponse.body()).getString("token");